src/lib/ERC4337/validator/TurnkeyValidator.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Validator} from "src/lib/ERC4337/validator/Validator.sol";
+import {UserOperation} from "src/lib/ERC4337/utils/UserOperation.sol";
+import {Operations} from "src/lib/Operations.sol";
+import {Access} from "src/access/Access.sol";
+import {SignatureChecker} from "openzeppelin-contracts/utils/cryptography/SignatureChecker.sol";
+
+/// @dev Validator module that restricts valid signatures to only come from recognized Turnkeys
+/// for the calling Accounts contract 
+contract TurnkeyValidator is Validator {

```

Comment:
This pattern is not specific to Turnkey, so let's rename. I think `CallPermitValidator` is apt because we need to check if the signer has the `CALL_PERMIT` permission

Reply:
Resolved in 4d4490a